sesman/chansrv/chansrv_config.c

```diff
@@ -266,18 +272,28 @@ config_read(int use_logger, const char *sesman_ini)
     return cfg;
 }
 
+/******************************************************************************/
+static const char *
+bool2text(int value)
```

Comment:
I'm fine with this at the moment but this function can be used in general contexts not only `chansrv`.
I think somewhere in `common/` will be a more appropriate place.  It's not a `os_calls` though.

Reply:
Absolutely, and there's one in log.c. I've moved it to the new string_calls.c file. We could maybe move some other stuff out of os_calls.c into here too. What do you think?

Reply:
I agree.